service/src/main/java/com/commercetools/pspadapter/payone/domain/ctp/CustomTypeBuilder.java

```diff
@@ -32,6 +32,8 @@
     public static final String PAYMENT_CREDIT_CARD = "payment-CREDIT_CARD";
     public static final String PAYMENT_WALLET = "payment-WALLET";
     public static final String PAYMENT_BANK_TRANSFER = "payment-BANK_TRANSFER";
+    public static final String PAYMENT_BANK_TRANSFER_IDEAL = "payment-BANK_TRANSFER_IDEAL";
```

Comment:
Why bank transfer? Is it like this in the docs?

Reply:
both new payment types that we are going to implement are bank transfers. This variable value is persisted in payment object but not passed down to payone payment gateway. It's just a name I came up with and has no influence on any factors.

Reply:
OK, but naming is important :) You could name it also CREDIT_CARD_IDEAL and it's also correct from the code perspective :) How do you know they're bank transfers? Is it in the docs as bank transfers?

Reply:
This is a quote from Specification Shop Integrations pdf file that Thomas Alf has attached to the first opening JIRA issue.

>For instance, the online bank transfer type iDeal is only available in the Netherlands and the shop integrator or merchant has to ensure that it only works with that country.

So, it stated iDeal as *onlilne bank transfer*. For bancontact it is similarly. Because both of these payment type requests require ** onlinebanktransfertype** parameter to be passed in the requests to payone.